mappings/net/minecraft/client/font/FontManager.mapping

```diff
@@ -1,8 +1,14 @@
 CLASS net/minecraft/class_378 net/minecraft/client/font/FontManager
 	FIELD field_18215 resourceReloadListener Lnet/minecraft/class_3302;
-	FIELD field_2259 textRenderers Ljava/util/Map;
+	FIELD field_2259 fontStorages Ljava/util/Map;
 	FIELD field_2260 textureManager Lnet/minecraft/class_1060;
 	FIELD field_2261 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_24254 MISSING_STORAGE_ID Lnet/minecraft/class_2960;
+	FIELD field_24255 missingStorage Lnet/minecraft/class_377;
+	FIELD field_24256 fontIdMapping Ljava/util/Map;
 	METHOD <init> (Lnet/minecraft/class_1060;)V
 		ARG 1 manager
 	METHOD method_18627 getResourceReloadListener ()Lnet/minecraft/class_3302;
+	METHOD method_27539 createTextRenderer ()Lnet/minecraft/class_327;
+	METHOD method_27541 setFontIdMapping (Ljava/util/Map;)V
```

Comment:
this is really a remapping, don't think `mapping` is a bad name here

Reply:
Just calling it a `mapping` doesn't give much clarity to what it's actually mapping -- we already know it's a mapping because the parameter type is a `Map`. What it actually does for the game is overrides fonts. In vanilla, this is what's used to force the Unicode font in `createTextRender`. Take a look at how it's used in `MinecraftServer#initText` as well.

Reply:
Done